aci/data_source_aci_fvrsprovcons.go

```diff
@@ -17,45 +17,44 @@ func dataSourceAciContractProvider() *schema.Resource {
 
 		SchemaVersion: 1,
 
-		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+		Schema: map[string]*schema.Schema{
 			"application_epg_dn": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
 			},
-
-			"contract_name": &schema.Schema{
+			"contract_dn": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
 			},
```

Comment:
I think we should keep contract_name as a deprecated option but an option and required either contract_name and contract_dn so we don't introduce a breaking change.

Reply:
The contract_name has been added with a deprecated warning message.